meta-lmp-base/recipes-support/compose-apps-early-start/compose-apps-early-start.bb

```diff
@@ -25,3 +25,4 @@ do_install() {
 }
 
 FILES:${PN} += "${systemd_system_unitdir}/*.service"
+RDEPENDS:${PN} = "aktualizr-lite-apps"
```

Comment:
Is this added mistakenly? If not, then maybe should be in dedicated commit.

Reply:
Removed.